package pass;

import java.lang.Exception;

public class TryCatchFinally {
    public void tryCatchFinally () {
        try {
            int x = 5;
        } catch (Exception e) {
            int x = 10;
        }
    }
}